src/report/mermaid/config.vsh.yaml

```diff
@@ -23,21 +20,48 @@ functionality:
       description: Generated network as output.
       direction: output
       required: true
+    - name: "--output_format"
+      type: string
+      description: |
+        Output format for the generated image. By default will be inferred from the extension 
+        of the file specified with --output.
+      choices: ["svg", "png", "pdf"]
+    - name: "--width"
+      type: integer
+      description: "Width of the page"
+      default: 800
+    - name: "--height"
+      type: integer
+      description: "Height of the page"
+      default: 600
+    - name: "--background_color"
+      type: string
+      description: "Background color for pngs/svgs (not pdfs)"
+      default: "white"
+      example: ["#F0F0F0"]
   resources:
-    - type: python_script
-      path: script.py
+    - type: bash_script
+      path: script.sh
+    - path: ./puppeteer-config.json
   test_resources:
     - type: python_script
       path: test.py
```

Comment:
```suggestion
    - type: bash_script
      path: test.sh
```

Reply:
I ran into issues with the component not being build before running the tests, and `viashpy` solves this issue so I did not want to change this :) But I guess this test is simple enough to just convert to bash